src/scripts/main.js

```diff
@@ -2,4 +2,41 @@
 
 document.addEventListener('click', (e) => {
   // write code here
+  const wall = document.querySelector('.wall');
+  const spider = document.querySelector('.spider');
+
+  wall.addEventListener('click', () => {
+    const wallRect = wall.getClientRects();
```

Comment:
The `getClientRects()` method returns a collection of DOMRect objects, which is not suitable for accessing `left` and `top` properties directly. You should use `getBoundingClientRect()` instead to get a single DOMRect object.